operator/pkg/securedcluster/scanner/auto_sense_test.go

```diff
@@ -18,16 +19,25 @@ var securedCluster = platform.SecuredCluster{
 	},
 }
 
+var validClusterVersion = &openshiftConfigv1.ClusterVersion{
+	ObjectMeta: metav1.ObjectMeta{
+		Name: "version",
+	},
+	Spec: openshiftConfigv1.ClusterVersionSpec{
+		ClusterID: "test-cluster-id",
+	},
```

Comment:
I'd drop this `spec` part here and below, unless I'm missing something and it is in fact required?

Reply:
It's actually worked without `ClusterID` even though both Spec and ClusterId marked as required but I moved tests to use unstructured as well so there is no longer such key anyway

Reply:
FYI the "required" is only enforced by the k8s API server, which is not used in these tests.